Wikipedia/Code/DiffController.swift

```diff
@@ -111,343 +109,53 @@ class DiffController {
     }
     
     private func fetchDiff(fromRevisionId: Int, toRevisionId: Int, theme: Theme, traitCollection: UITraitCollection, completion: @escaping ((Result<[DiffListGroupViewModel], Error>) -> Void)) {
+
+//        let queue = DispatchQueue.global(qos: .userInitiated)
```

Comment:
Wikidiff2 still needs to be deployed to output diff item offsets. Uncomment here to see it working with an embedded test response that is fully populated.